lib/tasks/correct_rent_type_value.rake

```diff
@@ -0,0 +1,20 @@
+desc "Alter rent_type values for bulk uploaded lettings logs for 2024 where they were not mapped correctly"
+task correct_rent_type_value: :environment do
+  affected_uploads = BulkUpload.where(log_type: "lettings", year: 2024)
+  affected_uploads.each do |upload|
+    upload.logs.where.not(rent_type: nil).each do |log|
+      current_rent_type = log.rent_type
+      rent_type_at_upload = log.versions.length == 1 ? log.rent_type : log.versions.first.next.reify.rent_type
+      next unless rent_type_at_upload == current_rent_type
+
+      new_rent_type_value = BulkUpload::Lettings::Year2024::RowParser::RENT_TYPE_BU_MAPPING[rent_type_at_upload]
+      log.rent_type = new_rent_type_value
```

Comment:
Is there any risk of this change causing other data to be cleared? (I think probably not - not sure anything depends on rent type, just want to be sure)

Reply:
I'm like 95% sure it should be fine. I couldn't find things dependent on the `rent_type` or `lettype`, we also get `lar` and `irproduct` from `rent_type`, which don't look like should be an issue. There's a conditional on `irproduct_other` but only if it's a 5. I don't think we'll be updating any `rent_type` to a 5 cause there was no way to set it to 6 in the first place.

We do also derive `renttype` which is used for tenancy length validation (and routing, but routing is only a different wording, still same question), which from what I checked before writing this, none of the logs should have failed.

The tests also check that the logs don't go from completed to in progress, but obviously, that's with a very specific combination of answers.